projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -498,6 +444,73 @@
 	}
 }
 
+.jetpack-field-multiple:where(.is-style-button) {
```

Comment:
Should this also be in a `style.scss?` so it has affect on the frontend?

Or is it in https://github.com/Automattic/jetpack/blob/update/specificity-of-form-styles-to-0-1-0/projects/packages/forms/src/contact-form/css/grunion.css#L697?

I'm seeing my global styles being overwritten in the editor:


<img width="400" alt="Screenshot 2025-04-22 at 4 40 58 pm" src="https://github.com/user-attachments/assets/ca3ed2bb-f63d-4058-9555-303e3b3f68ad" />


In the frontend:

<img width="400" alt="Screenshot 2025-04-22 at 4 41 16 pm" src="https://github.com/user-attachments/assets/000a0af3-21d8-4198-bb40-d5b251b42af5" />

Reply:
> Should this also be in a style.scss? so it has affect on the frontend?

I think Dan only had time to look at the editor styles in this PR at the end of last week, but my understanding is that the styles will need to be updated in the big `grunion.css` file which is a bit more fiddly than the usual `editor.scss` / `style.scss` approach that we're used to in blocks.

Reply:
If I had read the description, I would have known. 🤦🏻  

> This PR only addresses editor styles, the frontend styles still need to be updated.

Thank you!

Then it's working as described 👍🏻